### PR TITLE
added new strings for ru translation

### PR DESCRIPTION
**Pull request by X-NicON**
upd translation

### CODE REVIEW

File: res/values-ru/strings.xml [hunk 1]
```diff
@@ -312,7 +312,6 @@
     <string name="torrent_max_uploads">Максимум раздач</string>
     <string name="torrent_max_total_connections_summary">Максимальное количество подключений</string>
     <string name="torrent_max_total_connections">Максимальное количество подключений</string>
-    <string name="torrent_max_torrent_connection_summary">Максимум торрент соединений</string>
```
Comment (gubatron) [inline res/values-ru/strings.xml]: how come these are gone?
if gone, then this will appear in english.

File: res/values-ru/strings.xml [hunk 2]
```diff
@@ -384,7 +383,6 @@
     <string name="support_frostwire_summary">Показать объявления на поддержку развития этого приложения.</string>
     <string name="storage_preference_title">Выбор места хранения</string>
     <string name="storage_preference_summary">Место хранения по умолчанию, где будут сохранены файлы.</string>
-    <string name="connectivity">Соединение</string>
```
Comment (gubatron) [inline res/values-ru/strings.xml]: ?

**Comment by gubatron** [comment]
work in progress?

File: res/values-ru/strings.xml (hunk 1 above)
Comment (X-NicON) [inline res/values-ru/strings.xml]: they are absent in english "strings.xml"

**Comment by X-NicON** [comment]
Right now merged values and values-ru. I do not know yet whether the time for translation.

File: res/values-ru/strings.xml (hunk 1 above)
Comment (gubatron) [inline res/values-ru/strings.xml]: :+1:

File: res/values-ru/strings.xml [hunk 3]
```diff
@@ -476,4 +473,50 @@
     <string name="file_descriptor_sd_mounted">Этот файл был удален с SD-карты, или вы вставили другую SD-карту.</string>
     <string name="file_descriptor_sd_unmounted">Этот файл находиться на SD-карте, пожалуйста вставьте её.</string>
     <string name="help_translate_frostwire">Помогите перевести FrostWire</string>
+    <string name="use_tpb_summary">TPB is magnet based, works only on Wi-Fi</string>
```
Comment (gubatron) [inline res/values-ru/strings.xml]: let's have all these ones translated and then do the merge.

**Comment by X-NicON** [comment]
there was a time. did the translation. rebase?

**Pull request merged**